ember-toucan-core/src/components/form/error.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="type-xs-tight text-critical flex items-center mt-1" ...attributes>
+  {{! Icon taken from Tony's open source icon set, this is temporary! }}
```

Comment:
temporary as in this will be updated in this PR?

Reply:
Excellent question!  Still _very_ temporary/WIP, but not worth blocking the PR on so we can continue to drive ahead from a development point of view.  We should have a much better understanding from a design perspective of _exactly_ what we want in about 2-4 weeks if I had to guess, but for the time being this is essentially a placeholder.  

We know we want an icon of sorts here, but not quite sure what it should be yet or exactly how it should look.  We think something like this works, but we definitely want to validate.  And it could be removed completely if we find it doesn't work!  Around the time mentioned above as well, we'll probably have quite a few PRs going up around styling tweaks to match what our design system expects; however, since we're re-thinking Form elements completely, things are still up in the air a bit.  We should land safely soon™ though 😄 .  We're definitely taking advantage of being pre-release 😂 !